docs/source/usersguide/input.rst

```diff
@@ -1264,6 +1264,68 @@ The ``<tally>`` element accepts the following sub-elements:
           two post-collision energy bins will be created, one with energies
           between 0 and 1 MeV and the other with energies between 1 and 20 MeV.
 
+        :mu:
+          A monotonically increasing list of bounding **post-collision**
+          cosines of the change in a particle's angle (i.e.,
+          :math:`\mu = \cos(\Omega \cdot \Omega')`),
```

Comment:
I spoke too fast -- the dot product is already a cosine, so cosine around it is redundant. Sorry!